app/history_meta.py

```diff
@@ -53,7 +53,12 @@ def _col_copy(col):
         col = col.copy()
         orig.info['history_copy'] = col
         col.unique = False
-        col.default = col.server_default = None
+
+        # if the column is nullable, we could end up overwriting an on-purpose null value with a default.
```

Comment:
These sort of hacks are a bad smell. the sooner we get this versioning plugin out the better.